SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -112,6 +112,7 @@ const Worker_ComponentId VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID        = 9979;
 const Worker_ComponentId CLIENT_ENDPOINT_COMPONENT_ID					= 9978;
 const Worker_ComponentId SERVER_ENDPOINT_COMPONENT_ID					= 9977;
 const Worker_ComponentId MULTICAST_RPCS_COMPONENT_ID					= 9976;
+const Worker_ComponentId SPATIAL_DEBUGGING_COMPONENT_ID					= 9975;
```

Comment:
whitespace looks off in HG for me 
![image](https://user-images.githubusercontent.com/9222722/72982940-25862500-3dd8-11ea-8bf0-4097608b229e.png)

Reply:
It was lining up in Visual Studio, but I've re-whitespaced it to be sure.